desktop/main.dev.ts

```diff
@@ -61,7 +61,7 @@ let browserView: BrowserView;
 let tray: Tray;
 let nodeManager: NodeManager;
 let notificationManager: NotificationManager;
-let isDarkMode: boolean = nativeTheme.shouldUseDarkColors;
+const isDarkMode: boolean = StoreService.get('userSettings.darkMode') as boolean;
```

Comment:
Who using this variable?

Reply:
https://github.com/spacemeshos/smapp/blob/9378d964e9a3fb972066b704d27ef8b363cb0c53/desktop/main.dev.ts#L152

Reply:
I think we don’t need this `const isDarkMode` since it may lose actuality when we open a browser view (consumer of the variable). So I propose to get rid of this variable and call `StoreService.get('userSettings.isDarkMode')` instead.

Scenario:
1. User opens App (with default dark mode)
2. User opens Settings and switched to light mode.
3. User opens "Dashboard"

Expected: light dashboard
Actual: User will see the dark dashboard because on the App startup the `const isDarkMode` was set to `light`.

Reply:
This is a use case for first time user. How about existing user?

Reply:
If he will switch the theme and then open the Dashboard — it doesn't matter if it's the first time or not.
The fact is that this constant may become obsolete.
Just get rid of constant and ask a StoreService for an actual value when it needed.